main.go

```diff
@@ -33,6 +36,10 @@ func main1() int {
 	var err error
 	switch kingpin.MustParse(app.Parse(os.Args[1:])) {
 	case gen.FullCommand():
+		log.PrintCommands = *genDebugCommands
+		log.PrintPackageGenerated = *genDebugPackageGenerated
+		// Currently genDebugPackageGenerated doubles as the verbose flag
+		log.Verbose = *genDebugPackageGenerated
```

Comment:
If we were using `flag`, you could avoid this extra boilerplate via `flag.BoolVar(&log.PrintCommands, ...`. I'm sure that kingpin has something similar.

Reply:
I don't understand how what you are suggesting has less boilerplate than what is already there?

Reply:
Instead of:

```
someFlag = gen.Flag(...).Bool()
[...]
log.SomeFlag = *someFlag
```

You can just do

```
gen.Flag(...).BoolVar(&log.SomeFlag)
```

Of course you'd need to collapse `log.PrintPackageGenerated` into `log.Verbose`, which I think we should do anyway. For example, `go test -v` does more than print the packages being tested - it also prints extra info about the tests being run.

Reply:
Ah, gotcha! Done.